evergreen.yml

```diff
@@ -7,7 +7,6 @@ buildvariants:
       - name: unit_tests_3.9
       - name: unit_tests_3.10
       - name: unit_tests_3.11
-      - name: check_pypi_version
```

Comment:
I think we will still need this task which is to check the new version that we are publishing. Can you bump the version in [pyproject.toml](https://github.com/evergreen-ci/git-co-evg-base/blob/main/pyproject.toml#L3) and [CHANGELOG.md](https://github.com/evergreen-ci/git-co-evg-base/blob/main/CHANGELOG.md)? I believe this should be a major version bump as the change is not backward compatible.

Reply:
Got it. I think @rongodb said this check was removed in a different repo and we thought we could remove it here. Will look into it.

Reply:
Now I start worrying about that repo. Do we remember which one? @rongodb

Reply:
Ron sent me to [this slack thread](https://mongodb.slack.com/archives/C05UYTJNLUW/p1713203041206859) about the issue but can't seem to locate a PR where it was removed.

Reply:
https://github.com/evergreen-ci/evergreen.py/commit/979bb71232658344b1ec4ff1b9a0261dbab32aba

Reply:
Found the PR: https://github.com/evergreen-ci/evergreen.py/commit/979bb71232658344b1ec4ff1b9a0261dbab32aba
@zituo-jin

Reply:
We should have it back because it's to check a versions has already existed in PyPi, which can avoid deployment failure after PR is merged like [this one](https://spruce.mongodb.com/task/evergreen.py_ubuntu1804_deploy_da185a1e4988b4c760506831cd3a3ed3b88ddc6d_24_05_28_21_27_26/logs?execution=0) in the future (I missed that conversation as I was OOO).

Reply:
This task would make sure people update the version as mentioned in [versioning doc](https://github.com/evergreen-ci/evergreen.py/?tab=readme-ov-file#versioning). @llazala000 @rongodb

Reply:
I updated the changelog file.

Reply:
Filed [DEVPROD-8040 - Re-implement pypi-version-check in 10gen org](https://jira.mongodb.org/browse/DEVPROD-8040) as a follow-up.